draft-packages/button/KaizenDraft/Button/decision-tokens.scss

```diff
@@ -0,0 +1,24 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// Zen theme
+$dt-color-background-active: $kz-color-wisteria-100;
+$dt-color-background-hover: $kz-color-wisteria-100;
+$dt-color-background-focus: $kz-color-wisteria-100;
+$dt-color-border-color-base: $kz-color-wisteria-500;
+$dt-color-border-color-hover: $kz-color-wisteria-700;
+$dt-color-border-color-focus: $kz-color-wisteria-700;
+$dt-color-border-color-disabled: $kz-color-wisteria-500;
+$dt-color-color-disabled: $kz-color-wisteria-800;
+
+// Heart theme overrides
+$dt-color-background-active: var(--kz-color-stone, $kz-color-wisteria-100);
+$dt-color-background-focus: var(--kz-color-stone, $kz-color-wisteria-100);
+$dt-color-background-hover: var(--kz-color-stone, $kz-color-wisteria-100);
+$dt-color-border-color-base: var(--kz-color-iron-500, $kz-color-wisteria-500);
+$dt-color-border-color-hover: var(--kz-color-slate-700, $kz-color-wisteria-700);
+$dt-color-border-color-focus: var(--kz-color-slate-700, $kz-color-wisteria-700);
+$dt-color-border-color-disabled: var(
+  --kz-color-iron-500,
+  $kz-color-wisteria-500
+);
+$dt-color-color-disabled: var(--kz-color-wisteria-800, $kz-color-wisteria-800);
```

Comment:
I haven't checked how correct these are, I'm trusting you'll design review with @katsambarcus

Reply:
I've copy-pasta'd from here: https://github.com/cultureamp/kaizen-design-system/pull/972